builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -1052,3 +1051,46 @@ def getBuildModuleUrl() {
     return "http://${repo_base}/scm/${repo_core}/jazz-build-module.git"
   }
 }
+
+def handleDeploymentErrors(deployOutput, serviceName, env) {
+	if(deployOutput.contains('ROLLBACK_COMPLETE state and can not be updated')){
+		deleteAndRedeployService(env, serviceName)
+	}
+	else if (deployOutput.contains("CloudFormation - CREATE_FAILED - AWS::Logs::LogGroup - HandlerLogGroup"))
+	{
+		def lambda = "/aws/lambda/${serviceName}-${env}"
+		if (isExistingLogGroup(lambda)){
+			echo "Exception occured while serverless deploy : $logGroupErrString, Removing existing log group"
+			removeLogGroup(lambda)
+			deployOutput = deleteAndRedeployService(env, serviceName);
+		} else {
+			error "Unable to find existing log groups"
+		}
+	} 
+	else {
+			error "Exception occured while serverless deployment to ${env} environment"
+	}
+}
+
+def isExistingLogGroup (groupName){
+	def isExistingLogGrp = false
+	def filterLogs = sh (
+		script: "aws logs describe-log-groups --log-group-name-prefix $groupName --output json --profile cloud-api",
+		returnStdout: true
+	).trim()
+	def toJSON = new groovy.json.JsonSlurper()
+	def filterLogsJSON = toJSON.parseText(filterLogs)
+	def logGroups = filterLogsJSON.logGroups
+	echo "logGroups : $logGroups"
+	if(logGroups.size() > 0  && logGroups[0].logGroupName == groupName){
+		isExistingLogGrp = true
+	}
+	return isExistingLogGrp
+}
+
+def removeLogGroup(groupName) {
+	echo "Removing log group $groupName"
+	sh "aws logs delete-log-group --log-group-name $groupName --profile cloud-api"
```

Comment:
Use $credsId as the profile name (pass it as function param).

Reply:
Made the changes